dockers/Screenshotter/screenshotter.js

```diff
@@ -27,74 +27,32 @@ const newDir = path.normalize(
-        help: "Full URL of the Selenium web driver",
-    })
-    .option("seleniumIP", {
-        full: "selenium-ip",
-        help: "IP address of the Selenium web driver",
-    })
-    .option("seleniumPort", {
-        full: "selenium-port",
-        "default": 4444,
-        help: "Port number of the Selenium web driver",
-    })
-    .option("katexURL", {
-        full: "katex-url",
-        help: "Full URL of the KaTeX development server",
-    })
-    .option("katexIP", {
-        full: "katex-ip",
-        help: "Full URL of the KaTeX development server",
-    })
-    .option("katexPort", {
-        full: "katex-port",
-        help: "Port number of the KaTeX development server",
-    })
-    .option("include", {
-        abbr: "i",
-        help: "Comma-separated list of test cases to process",
-    })
-    .option("exclude", {
-        abbr: "x",
-        help: "Comma-separated list of test cases to exclude",
-    })
-    .option("reload", {
-        flag: true,
-        help: "Reload page for each test",
-    })
-    .option("verify", {
-        flag: true,
-        help: "Check whether screenshot matches current file content",
-    })
-    .option("diff", {
-        flag: true,
-        help: "With `--verify`, produce image diffs when match fails",
-    })
-    .option("new", {
-        flag: true,
-        help: "With `--verify`, generate new screenshots when match fails",
-    })
-    .option("attempts", {
-        help: "Retry this many times before reporting failure",
-        "default": 5,
-    })
-    .option("wait", {
-        help: "Wait this many seconds between page load and screenshot",
-    })
-    .parse();
+const opts = require("commander")
+    .option("-b, --browser <firefox|chrome>",
+        "Name of the browser to use (default firefox)")
+    .option("-c, --container <id>",
+        "Name or ID of a running docker container to contact")
+    .option("--selenium-url <url>", "Full URL of the Selenium web driver")
+    .option("--selenium-ip <ip>", "IP address of the Selenium web driver")
+    .option("--selenium-port <n>",
+        "Port number of the Selenium web driver (default 4444)")
+    .option("--katex-url <url>", "Full URL of the KaTeX development server")
+    .option("--katex-ip <ip>", "IP address of the KaTeX development server")
+    .option("--katex-port <n>", "Port number of the KaTeX development server")
+    .option("-i, --include <tests>",
+        "Comma-separated list of test cases to process")
+    .option("-x, --exclude <tests>",
+        "Comma-separated list of test cases to exclude")
+    .option("--reload", "Reload page for each test")
+    .option("--verify", "Check whether screenshot matches current file content")
+    .option("--diff", "With `--verify`, produce image diffs when match fails")
+    .option("--new",
+        "With `--verify`, generate new screenshots when match fails")
+    .option("--attempts <n>",
+        "Retry this many times before reporting failure (default 5)")
```

Comment:
Shouldn't `--selenium-port`, `--katex-port`, and `--attempts` be collected as an integer(`parseInt`)?